samples/cloudwatch/README.md

```diff
@@ -1,73 +1,88 @@
 # Amazon CloudWatch Integration
 
 This sample describes how to configure Amazon CloudWatch to collect metrics
-from the services and components in SAS Viya. Completing the deployment
-and configuration steps here will enable Amazon CloudWatch to collect and
-visualize the metrics exposed by a SAS Viya deployment as well as the logging
-components of viya4-monitoring-kubernetes.
+from the services and components in SAS Viya. Complete the deployment
+and configuration steps in this document to enable Amazon CloudWatch to collect and
+visualize metric information from both SAS Viya as well as the monitoring components deployed from this repository. This sample currently does not provide information for using Amazon applications to view SAS Viya log messages.
 
-The deployment and configuration of the non-Prometheus CloudWatch Agent is out
-of scope for this sample. Information on the default CloudWatch Agent is
-covered extensively in the online
-[CloudWatch documentation](https://aws.amazon.com/cloudwatch/getting-started/).
+## CloudWatch Agent
+
+AWS uses the CloudWatch agent to collect metrics. There are two types of CloudWatch agents that you might want to use in your environment:
+
+- The CloudWatch agent with Prometheus monitoring scrapes metrics from Prometheus sources, such as SAS Viya. The metrics can then be converted and mapped for display in CloudWatch. Because SAS Viya monitoring uses Prometheus, this sample explains how to deploy and configure only the CloudWatch agent for use with Prometheus. 
+
+- The default CloudWatch agent collects metrics from AWS nodes. Although these metrics are not specific to SAS Viya, you might want to use both agents so you can obtain a complete view of your environment's performance. For information about using the default agent, see the 
+[CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html). If you do use the default agent, you must add tolerations to the
+agent's daemonset definition so that the agent can run on nodes that are tainted to support SAS Viya workload node placement. See [CloudWatch Agent Daemonset and Workload Node Placement](#wnp_tolerations).
```

Comment:
Should we make it clear that this is pointer to elsewhere in the same document? I wasn't sure where the link was going to take and assumed it was a whole separate document.  Perhaps: _See the [blah blah] section below for more information._

Reply:
It's normal to link to sections of a document. Not sure what the issue is here. Click link, get info - regardless of whether it's on the same page or not.

Reply:
I agree with Bryan - I don't think a notice is needed that the link goes to the same document.